### PR TITLE
owner, metrics(ticdc): fix metrics

### LINKED ISSUE #10438: Add create time metric for every changefeed
### Is your feature request related to a problem?

No

### Describe the feature you'd like

I need a create time metric for every changefeed to record how long does a changefeed live since being created.

### Describe alternatives you've considered

NA

### Teachability, Documentation, Adoption, Migration Strategy

With create time metric, 1. we can reduce lag delay alarms noise when creating a new changefeed; 2. It's also a important information help trouble shooting some other problems

### LINKED ISSUE #10447: [CDC] Distinguish dataflow metrics between incremental scan and normal raft logs
image

Currently, we only have a total output events. But the events are consist by two types: incremental scan and raft logs. We need to distinguish between them.

### LINKED ISSUE #10449: ticdc not delete ticdc_owner_status metrics when removing changefeed
reproduce steps
1. create a changefeed
2. pause the changefeed
3. remove the changefeed
4. curl http://localhost:8300/metrics , and that metrics about the removed changefeed still exist.

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?

Issue Number: close #10449, close #10447, close #10438

### What is changed and how it works?
1. For #10449, always cleanup metrics [whether the changefeed has been cleaned or not](https://github.com/pingcap/tiflow/pull/10459/files#diff-953746562776cf8aa41cd4a971df8c546649d2721ae787607aed2069f3b44eb0R732-L759).
image

2. For #10447, use the type of ticdc_kvclient_pull_event_count to distinguish incresmental scan and real-time raft log.
image

3. For #10438, add changefeedStartTimeGauge metric, which record the changefeed create time and restart time. **Note that** changefeedStartTimeGauge records the physical time with ***millisecond precision.***
image

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by hicqu** [comment]
For grafana changes, could you show a picture for more details?

**Review by grovecai — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

File: cdc/owner/changefeed.go [hunk 1]
```diff
@@ -794,6 +804,8 @@ func (c *changefeed) cleanupMetrics() {
 
 	if c.isRemoved {
 		changefeedStatusGauge.DeleteLabelValues(c.id.Namespace, c.id.ID)
+		changefeedCheckpointTsGauge.DeleteLabelValues(c.id.Namespace, c.id.ID, "create")
+		changefeedCheckpointTsLagGauge.DeleteLabelValues(c.id.Namespace, c.id.ID, "restart")
```
Comment (kennytm) [inline cdc/owner/changefeed.go:808]: 🤔 should these two be deleting label values from `changefeedStartTimeGauge` instead?

**Pull request merged**